### PR TITLE
cleaning and conversion to package format 2

### PR DESCRIPTION
**Pull request by ipa-rmb**
should solve #36 and #38

### CODE REVIEW

**Comment by ipa-rmb** [comment]
Please fix travis, the *_data packages are only declared as exec_dependencies (and everything builds fine on my system without the *_data packages).

**Comment by fmessmer** [comment]
Travis fails because there is no rosdep key for the *__data package...we might need to add it to rosdistro? or completely remove (even exec_) dependencies to those packages...

File: cob_fiducials/package.xml [hunk 1]
```diff
@@ -1,47 +1,32 @@
...
-        <run_depend>std_srvs</run_depend>
-        
-        <run_depend>tinyxml</run_depend>
-        <run_depend>boost</run_depend>
-        <run_depend>libpcl-all</run_depend>
+		<description>
+			Fiducial recognition. Implementation of different 2D tags like PI-tag from Bergamasco et al. for recognition with a single 2D camera.
+		</description>
+		<author>Jan Fischer</author>
+		<maintainer email="jsf@ipa.fhg.de">Jan Fischer</maintainer>
```
Comment (fmessmer) [inline cob_fiducials/package.xml:9]: change maintainer

File: cob_fiducials/package.xml [hunk 2]
```diff
@@ -1,47 +1,32 @@
...
+		<depend>image_transport</depend>
+		<depend>cv_bridge</depend>
+		<depend>visualization_msgs</depend>
+		<depend>cob_vision_utils</depend>
+		<depend>cob_object_detection_msgs</depend>
+		<depend>std_srvs</depend>
+		
+		<depend>tinyxml</depend>
+		<depend>boost</depend>
+		<depend>libpcl-all-dev</depend>
```
Comment (fmessmer) [inline cob_fiducials/package.xml:29]: alphabetic order

File: cob_marker/package.xml [hunk 3]
```diff
@@ -1,53 +1,35 @@
...
 
-  </description>
-  <author>Joshua Hampp</author>
-  <maintainer email="jsf@ipa.fhg.de">Jan Fischer</maintainer>
-  <version>0.1.0</version>
-  <license>LGPL</license>
-  <url>http://ros.org/wiki/cob_marker</url>
+	</description>
+	<author>Joshua Hampp</author>
+	<maintainer email="jsf@ipa.fhg.de">Jan Fischer</maintainer>
```
Comment (fmessmer) [inline cob_marker/package.xml:9]: change maintainer

File: cob_marker/package.xml [hunk 4]
```diff
@@ -1,53 +1,35 @@
...
-  <run_depend>tf_conversions</run_depend>
-  <run_depend>cob_object_detection_msgs</run_depend>
-  <run_depend>visualization_msgs</run_depend>
-  <run_depend>actionlib</run_depend>
-  <run_depend>image_transport</run_depend>
-  <run_depend>libzxing</run_depend>
+	<depend>libpcl-all-dev</depend>
+	<depend>boost</depend>
+	<depend>libdmtx-dev</depend>
+	<depend>libmagick</depend>
```
Comment (fmessmer) [inline cob_marker/package.xml:33]: alphabetic order

File: cob_object_categorization/package.xml [hunk 5]
```diff
@@ -1,52 +1,36 @@
...
+<package format="2">
+	<name>cob_object_categorization</name>
+	<version>0.1.0</version>
+	<description>
+
+		Image processing tool for the categorization differnt images objects
+
+	</description>
+	<author>Richard Bormann</author>
+	<maintainer email="jsf@ipa.fhg.de">Jan Fischer</maintainer>
```
Comment (fmessmer) [inline cob_object_categorization/package.xml:10]: change maintainer

File: cob_object_categorization/package.xml [hunk 6]
```diff
@@ -1,52 +1,36 @@
...
+	<depend>nodelet</depend>
+	<depend>cob_perception_msgs</depend>
+	<depend>cob_object_detection_msgs</depend>
+	<depend>tf</depend>
+	<depend>actionlib</depend>
+	<depend>actionlib_msgs</depend>
+
+	<depend>libpcl-all-dev</depend>
+	<depend>eigen</depend>
+	<depend>libvtk</depend>
```
Comment (fmessmer) [inline cob_object_categorization/package.xml:32]: alphabetic order

File: cob_object_detection_fake/package.xml [hunk 7]
```diff
@@ -1,22 +1,19 @@
...
+		fake implementation for detecting objects, in order to test other system components
 
-  </description>
-  <author>Joshua Hampp</author>
-  <maintainer email="jsf@ipa.fhg.de">Jan Fischer</maintainer>
-  <license>LGPL</license>
-  <url>http://ros.org/wiki/cob_object_detection_fake</url>
+	</description>
+	<author>Joshua Hampp</author>
+	<maintainer email="jsf@ipa.fhg.de">Jan Fischer</maintainer>
```
Comment (fmessmer) [inline cob_object_detection_fake/package.xml:10]: change maintainer

File: cob_object_detection_fake/package.xml [hunk 8]
```diff
@@ -1,22 +1,19 @@
...
-  <buildtool_depend>catkin</buildtool_depend>
+	<buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>rospy</build_depend>
-  <build_depend>cob_object_detection_msgs</build_depend>
-
-  <run_depend>rospy</run_depend>
-  <run_depend>cob_object_detection_msgs</run_depend>
+	<depend>rospy</depend>
+	<depend>cob_object_detection_msgs</depend>
```
Comment (fmessmer) [inline cob_object_detection_fake/package.xml:17]: alphabetic order

File: cob_object_perception/package.xml [hunk 9]
```diff
@@ -8,14 +8,16 @@
 	<maintainer email="jsf@ipa.fhg.de">Jan Fischer</maintainer>
```
Comment (fmessmer) [inline cob_object_perception/package.xml:8]: change maintainer

File: cob_object_recording/package.xml [hunk 10]
```diff
@@ -1,44 +1,31 @@
...
+<package format="2">
+	<name>cob_object_recording</name>
+	<version>0.1.0</version>
+	<description>
+
+		handy tool to record image data from an object from euqally spaces views. For its usage it is necesarry to print out a mrker board, which is available from the cob_fiducials package in the cob_object_perception meta-package
+
+	</description>
+	<author>Richard Bormann</author>
+	<maintainer email="jsf@ipa.fhg.de">Jan Fischer</maintainer>
```
Comment (fmessmer) [inline cob_object_recording/package.xml:10]: change maintainer

File: cob_object_recording/package.xml [hunk 11]
```diff
@@ -1,44 +1,31 @@
...
+	<depend>roslib</depend>
+	<depend>dynamic_reconfigure</depend>
+	<depend>cv_bridge</depend>
+	<depend>std_msgs</depend>
+	<depend>std_srvs</depend>
+	<depend>sensor_msgs</depend>
+	<depend>message_filters</depend>
+	<depend>image_transport</depend>
+	<depend>visualization_msgs</depend>
+	<depend>cob_object_detection_msgs</depend>
```
Comment (fmessmer) [inline cob_object_recording/package.xml:28]: alphabetic order

**Comment by fmessmer** [comment]
@ipa-rmb please do not continue development on this branch...your latest changes are not related to cleanup and format2 conversion anymore...use feature branches!

**Pull request merged**